openeo_udp/examples/max_ndvi_composite/generate.py

```diff
@@ -0,0 +1,118 @@
+import json
+import sys
+from pathlib import Path
+
+import openeo
+from attr.validators import optional
+from openeo.api.process import Parameter
+from openeo.processes import array_create
+from openeo.rest.udp import build_process_dict
+
+# TODO #15 where to put reusable helpers? e.g. load description from README.md, dummy openeo connection, properly write to JSON file, ...
+
+
+def generate() -> dict:
+    # TODO: use/inject dummy connection instead of concrete one? (See cdse_marketplace_services)
+    c = openeo.connect("openeofed.dataspace.copernicus.eu")
+
+    spatial_extent = Parameter.bounding_box(name="spatial_extent", default=None, optional=True)
+    temporal_extent = Parameter.temporal_interval(name="temporal_extent")
+    schema = {"type": "string", "enum":["B02","B03","B04","B05","B06","B07","B08","B8A","B11","B12"]}
+    bands_param = Parameter.array(name="bands",description="Sentinel-2 L2A bands to include in the composite.", item_schema=schema, default=["B04", "B03", "B02"], optional=True)
+
+    max_cloud_description = """
+    The maximum cloud cover percentage to filter Sentinel-2 inputs at full product level.
+    By reducing the percentage, fewer input products are considered, which also potentially increases the risk of missing valid data.  
+    We do not recommend setting it higher than 95%, as this decreases performance by reading very cloudy areas with little chance of finding good pixels.
+    
+    For composites over large time ranges, a reduced value can help to consider only good quality input products, with few undetected clouds.
+    
+    """
+    max_cloud_cover_param = Parameter.number(name="max_cloud_cover", description=max_cloud_description, default=75.0, optional=True)
+
+    scl = c.load_collection(
+        "SENTINEL2_L2A",
+        temporal_extent=temporal_extent,
+        bands=["SCL"],
+        max_cloud_cover=max_cloud_cover_param
+    ).resample_spatial(projection="EPSG:25832", resolution=10)
+
+    def scl_to_masks(bands):
+        scl_layer = bands[0]
+        to_mask = any(array_create([
```

Comment:
I would use `openeo.processes.any` here instead of just `any`, it's very easy to miss the `from openeo.processes import any` lower and think it's python's standard `any` at work here